pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1572,7 +1578,7 @@ func (og *operationGenerator) GenerateExpandVolumeFunc(
 			// k8s doesn't have transactions, we can't guarantee that after updating PV - updating PVC will be
 			// successful, that is why all PVCs for which pvc.Spec.Size > pvc.Status.Size must be reprocessed
 			// until they reflect user requested size in pvc.Status.Size
-			kubeClient := og.getKubeClient(pv.Tenant)
+			kubeClient := og.getDefaultKubeClient()
```

Comment:
Why default? It this used in RP only?

Reply:
this will assumed to be used in TP (as part of KCM) only. the default kube client is pointing to TP. see line 868-869 comment.

However, if this assumption is nt valid, this par tof code need to change.